setup.py

```diff
@@ -1,5 +1,31 @@
 from setuptools import setup, find_packages
 
+
+def union(*deps):
+    return list(set().union(*[extras[dep] for dep in deps]))
+
+extras = {
+    "envs": [
+        "atari_py~=0.2.0",            # atari environments
+        "box2d-py~=2.3.5",            # box2d environments
+        "pybullet>=3.0.6",            # continuous environments
+        "Pillow",                     # rendering library
```

Comment:
This is new? Also shouldn't it have a version?

Reply:
Here's the story: If you include both `gym` and `gym[atari,box2d]` as dependencies, apparently it only installs `gym`, so `pip install autonomous-learning-library[envs]` does not do what it is supposed to. To make it work properly I had to unroll the dependencies, which kind of stinks. `gym` did not have a version of Pillow specified, but I agree we should figure out a version number.